components/Primitives/samples/SwitchPresenter/SwitchConverterBrushSample.xaml

```diff
@@ -0,0 +1,45 @@
+<Page x:Class="PrimitivesExperiment.Samples.SwitchPresenter.SwitchConverterBrushSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:CommunityToolkit.WinUI.Controls"
+      xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:PrimitivesExperiment.Samples.SwitchPresenter"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:ui="using:CommunityToolkit.WinUI"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <!--
+            If you reference an enum directly in UWP, you need to use it somewhere for the XamlTypeInfo reference to be generated...
+        -->
+        <local:CheckStatus x:Key="MyChecks">Warning</local:CheckStatus>
+
+        <!--  SwitchConverter lets you easily convert general values to resources  -->
+        <!--  Note: This is in the converters namespace  -->
+        <converters:SwitchConverter x:Key="StatusToColorSwitchConverter"
+                                    TargetType="local:CheckStatus">
+            <!--  Note: These are reused from the controls namespace from SwitchPresenter  -->
+            <controls:Case Content="{ThemeResource SystemFillColorCriticalBrush}"
+                           Value="Error" />
+            <controls:Case Content="{ThemeResource SystemFillColorCautionBrush}"
+                           Value="Warning" />
+            <controls:Case Content="{ThemeResource SystemFillColorSuccessBrush}"
+                           Value="Success" />
+        </converters:SwitchConverter>
+    </Page.Resources>
+
+    <StackPanel Spacing="8">
+        <ComboBox x:Name="StatusPicker"
+                  Header="Pick a status"
+                  SelectedIndex="0">
+            <x:String>Error</x:String>
```

Comment:
Just looking at the sample in-app, maybe Success should go first (so it's default) and Error would go last? The comment "This is it, this is the demo" does well to clarify this, but the error might confuse some people.